src/sdk-reference/js/6/offline-tools/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: sdk.html.hbs
+title: Offline Tools
+description: Tools to handle the loss of connection to the Kuzzle server
+order: 400
+separator: essentials
+---
+
+# Offline tools
+
+The Kuzzle SDK provides a set of properties that help your application to be resilient to the loss of network connection
+during its lifespan. 
+
+## offlineQueue
+
+A read-only `Object` containing the requests queued while the SDK is in the `offline` state.
+
+## queueMaxSize
+
+A writable `number` defining the maximun size of the `offlineQueue`.
+
+## queueTTL
+
+A writable `number` defining the time in milliseconds a queued request is kept in the `offlineQueue`.
+
+## startQueuing()
+
+Starts the requests queuing. Request will be put in the `offlineQueue` instead of being discarded, until `stopQueuing` is called.
```

Comment:
```suggestion
Starts queuing requests when in `offline` state. Request will be put in the `offlineQueue` instead of being discarded, until `stopQueuing` is called.
```